net/route.go

```diff
@@ -55,3 +56,16 @@ func forEachRoute(ignoreIfaceNames map[string]struct{}, check func(r netlink.Rou
 	}
 	return nil
 }
+
+func AddRoute(link netlink.Link, scope netlink.Scope, dst *net.IPNet, gw net.IP) error {
+	err := netlink.RouteAdd(&netlink.Route{
+		LinkIndex: link.Attrs().Index,
+		Scope:     scope,
+		Dst:       dst,
+		Gw:        gw,
+	})
+	if os.IsExist(err) { // squash duplicate route errors
```

Comment:
I believe it was constructed empirically, from observing the "File exists" message from `ip route`, then examining the construction of `os.IsExist()`.  It seems better to depend on `os` rather than `syscall` as it is higher-level.

But this code has just been moved out of the CNI plugin; if we think it's wrong then that should be raised as a separate issue.